### PR TITLE
[5.2] Note on shrinkwrap

### PR DESCRIPTION
**Pull request by lucasmichot**
cc @franzliedke presentation on Laravel Elixir
PR in live from Laravel Meetup in Berlin

### CODE REVIEW

File: elixir.md [hunk 1]
```diff
@@ -50,6 +50,13 @@ Next, you'll want to pull in [Gulp](http://gulpjs.com) as a global NPM package:
 
     npm install --global gulp
 
+### Dependencies mamagment
+If you use a version control system, make sure to lock your NPM requirements:
```
Comment (GrahamCampbell) [inline elixir.md]: cs

**Comment by franzliedke** [comment]
Haha, awesome. You're a machine, @lucasmichot.

Thanks for coming by.

**Comment by lucasmichot** [comment]
Ah ah @franzliedke, it was a pleasure!
Preparing my slides for the next Meetup at the moment, with hopefully a less robotic voice
Have fun in Itaewon!

File: elixir.md [hunk 2]
```diff
@@ -50,6 +50,13 @@ Next, you'll want to pull in [Gulp](http://gulpjs.com) as a global NPM package:
 
     npm install --global gulp
 
+### Dependencies management
+If you use a version control system, make sure to lock your NPM requirements:
```
Comment (GrahamCampbell) [inline elixir.md]: missing newline

**Pull request merged**